eval/asserts/fix-loop-convergence.test.mjs

```
@@ -0,0 +1,210 @@
+
+## 実施順序
+
+| 順序 | 修正単位 | 変更対象 |
+|---|---|---|
+| 1 | FP-PICKER-ORDER | picker order setter |
+`;
+const E12_JUDGEMENT = 'JUDGEMENT: bw2_inherited=継承; '
+  + 'bw2_recurrence_confirmed=確認済み; bw2_cumulative=3; '
+  + 'nonactionable_in_work=含まない';
+const E12_CLAUDE_OPUS_OUTPUT = readFileSync(
+  new URL('../fixtures/fix-loop-convergence/e12-claude-opus-output.md', import.meta.url),
```

ESLint

[error] 'URL' is not defined.

(no-undef)